lib/home/profile/account/change_password.dart

```diff
@@ -1,17 +1,219 @@
 import 'package:flutter/material.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:thc/utils/navigator.dart';
```

Comment:
```suggestion
import 'package:thc/utils/local_storage.dart';
import 'package:thc/utils/navigator.dart';
import 'package:thc/utils/style_text.dart';
```

(imports that will enable the suggested changes further down)